### PR TITLE
[MFTF] Coverage for the 30645 Customer Edit Issue when there is a newsletter queue associated with it 

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
The test covers [the 30645 Customer Edit Issue when there is a newsletter queue associated with it ](https://github.com/magento/magento2/issues/30645)

### Related Pull Requests
[PR 30727](https://github.com/magento/magento2/pull/30727)

### Manual testing scenarios (*)
1. Create a customer
2. Create a newsletter template
3. Queue the newsletter template 
4. Update customer information

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminCreateNewsletterTemplateActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminCreateNewsletterTemplateActionGroup" extends="AdminMarketingCreateNewsletterTemplateActionGroup">
+        <annotations>
+            <description>Extends AdminMarketingCreateNewsletterTemplateActionGroup.
+                         Clicks the Show/Hide button for the Template Content field before text is sent to this field</description>
```
Comment (rogyar) [inline app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminCreateNewsletterTemplateActionGroup.xml]: Please, fix the indentation 

```suggestion
            <description>
                Extends AdminMarketingCreateNewsletterTemplateActionGroup. Clicks the Show/Hide button for the Template Content field before text is sent to this field
            </description>
```

File: app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminQueueNewsletterActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+
+    <actionGroup name="AdminQueueNewsletterActionGroup">
+        <annotations>
+            <description>Sends Newsletter template to queue:
```
Comment (rogyar) [inline app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminQueueNewsletterActionGroup.xml]: Let me shed some additional light using this example. 
In HTML/XHTML/XML the most commonly used practices in terms of such indentations are the following.

If the opening and closing tags are in the same line, we may put content between these tags.

```xml
<tag>Content</tag>
```

If we want to split lines, the common practice is to have the opening and closing tags in dedicated lines.

```xml
<tag>
    Some long long long long content
</tag>
```

So having

```xml
<tag> Some long long long long 
    and even more long content
</tag>
```

Is not a common practice.

I hope this example makes sense. 

Thank you!

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @AnnaAPak. Thank you for providing this useful refactoring. 
Please, check my comments below.

Thanks

File: app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminCreateNewsletterTemplateActionGroup.xml (hunk 1 above)
Comment (engcom-Hotel) [reply app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminCreateNewsletterTemplateActionGroup.xml]: Has been fixed.

File: app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminQueueNewsletterActionGroup.xml (hunk 2 above)
Comment (engcom-Hotel) [reply app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminQueueNewsletterActionGroup.xml]: Has been fixed.

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests CE, Static Tests, WebAPI Tests

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8823](https://jira.corp.magento.com/browse/ENGCOM-8823) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**